dms/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "13.0.8.0.0",
+    "version": "13.0.8.1.0",
```

Comment:
No need to bump versions - the Bot can do that when merging

Reply:
You need it if you want to test a migration script.